packages/blade/src/components/Alert/__tests__/Alert.web.test.tsx

```diff
@@ -108,6 +106,7 @@ describe('<Alert />', () => {
         />,
       ),
     ).toThrow(`[Blade: Alert]: SecondaryAction is allowed only when PrimaryAction is defined.`);
+    jest.restoreAllMocks();
```

Comment:
I think this would restore all the mocks no? even the ones that are not set inside this `it`.

I think so far we've been doing `afterAll` so should've been fine there. But if I add a `beforeAll` mock in this file now and write a test after this test block, I would imagine the mock would get restored before the next test runs which might not be expected 🤔 

Can we figure out a way to only restore `console.error` here?

Reply:
I think something like `console.error.mockRestore()` might make more sense no? because if someone puts a mock in `beforeAll`, then that will only work for first test then.

Reply:
Sounds good. @dev-cj could you please incorporate the suggestion from this thread?

Reply:
Yeah make sense, will implement it.

Reply:
New changes added.